.github/workflows/docker.yml

```
@@ -57,3 +57,31 @@ jobs:
 
       - run: docker push $DOCKER_IMAGE
         if: github.event_name == 'push'
+
+      - name: Generate token
+        if: github.event_name == 'push'
+        id: app-token
+        uses: getsentry/action-github-app-token@v1
+        with:
+          app_id: ${{ secrets.GA_OS_WORKFLOW_TRIGGER_APP_ID }}
+          private_key: ${{ secrets.GA_OS_WORKFLOW_TRIGGER_APP_PEM }}
+      
+      - name: Init variables
+        run: |
```

Maybe you can also add 
```suggestion
if: github.event_name == 'push'
```

---

Done in 1f714462.